src/onboarding/registration/ImportSelect.tsx

```diff
@@ -132,40 +127,35 @@ const styles = StyleSheet.create({
   cardTitle: {
     ...typeScale.labelMedium,
     color: colors.primary,
+    flex: 1,
```

Comment:
This flex is important so the text wraps instead of going into the card padding.